sicp/main.rkt

```diff
@@ -24,8 +25,8 @@
 (define+provide nil '())
 (define+provide the-empty-stream '())
 (define+provide stream-null? null?)
-(define+provide (inc x) (+ x 1))
-(define+provide (dec x) (- x 1))
+(define+provide inc add1)
```

Comment:
We intentionally do not do this because:

```
(define inc add1)
(println inc) ;=> #<procedure:add1>
```

prints a name that we do not want.

Reply:
I hadn't thought of that. I guess I can revert that part, then. What about the other changes?